packages/next/src/views/Version/RenderFieldsToDiff/utilities/getFieldsForRowComparison.ts

```diff
@@ -0,0 +1,60 @@
+import type { ArrayFieldClient, BlocksFieldClient, ClientField } from 'payload'
+
+import { getUniqueListBy } from 'payload/shared'
+
+/**
+ * Get the fields for a row in an iterable field for comparison.
+ * - Array fields: the fields of the array field, because the fields are the same for each row.
+ * - Blocks fields: the union of fields from the comparison and version row,
+ *   because the fields from the version and comparison rows may differ.
+ */
+export function getFieldsForRowComparison({
+  comparisonRow,
+  field,
+  versionRow,
+}: {
+  comparisonRow: any
+  field: ArrayFieldClient | BlocksFieldClient
+  versionRow: any
+}) {
+  let fields: ClientField[]
+
+  if (field.type === 'array' && 'fields' in field) {
+    fields = field.fields
+  }
+
+  if (field.type === 'blocks') {
+    fields = [
+      // {
```

Comment:
Need a little cleanup here?

Reply:
@DanRibbens thank you for the review! 🙏 

I had the same reaction when I saw this code which I extracted from the [Iterable/index.tsx](https://github.com/payloadcms/payload/blob/ef4b8d9b00d6f7de58a6a0f536100fa1ee66dfd6/packages/next/src/views/Version/RenderFieldsToDiff/fields/Iterable/index.tsx#L52-L56) component. It looked like it could have been a leftover, but then I though that it could have been left intentionally to inform the reader on the shape of blocks data. 

That being said, since we both had the same initial reaction I will remove it.